src/arch/riscv/pagetable_walker.cc

```diff
@@ -490,8 +490,29 @@ Walker::WalkerState::recvPacket(PacketPtr pkt)
             Addr paddr = walker->tlb->translateWithTLB(vaddr, satp.asid, mode);
             req->setPaddr(paddr);
             walker->pma->check(req);
+
+            if (pmode != PrivilegeMode::PRV_M){
+                bool pass = walker->pmp->pmp_check(req, mode);
```

Comment:
what does "pass" mean?

Reply:
"pass=true" means that the pmp checks are passed. Added a comment.